### DISCUSSION TITLE
WYSIWYG editor for restructured text?

### DISCUSSION THREAD

**Discussion by varna9000** [getnikola/nikola#3557] [General]
It's not Nikola specific, but I'l like to have an editor where I write the text with formatting and then save it as RST. I've looked at [Typora](https://typora.io), it's great, but it uses markdown by default, not RST.  Anyone knows RST wysywig editor?

**Comment by ralsina** [comment]
Sadly AFAIk there are no WYSIWYG editors for rst mostly because it's a
complicated format (for example, there is no way to programmatically modify
the parsed document and *save* it)
